R/cre.R

```diff
@@ -118,13 +118,13 @@ cre <- function(y, z, X, method_params, hyper_params){
   list2env(method_params, envir = environment())
   list2env(hyper_params, envir = environment())
 
-  # Split data -----------------------------------------------------------------
-  logger::log_info("Working on splitting data ... ")
+  # Honest Splitting -----------------------------------------------------------
+  logger::log_info("Honest Splitting")
```

Comment:
Same here. When reading the log file it is hard to understand what is happening.